packages/neuron-wallet/src/services/networks.ts

```diff
@@ -176,3 +183,17 @@ export default class NetworksService extends Store {
     return network
   }
 }
+
+function applyLocalhostIPv4Resolve(url: string): string {
```

Comment:
How about using `dns.setDefaultResultOrder('ipv4first')`

Reply:
Ah I see the discussion at https://github.com/Magickbase/neuron-public-issues/issues/122#issuecomment-1524625828

Reply:
> How about using `dns.setDefaultResultOrder('ipv4first')`

If we use this API, it is highly likely to become a technical debt because it may affect other DNS resolutions, which may lead us to make decisions to write other code with a bias towards using IPv4, while the future should embrace IPv6.

Reply:
Got it, please add this consideration in the issue https://github.com/Magickbase/neuron-public-issues/issues/122#issuecomment-1526892427